ff/src/fields/models/fp/montgomery_backend.rs

```diff
@@ -371,8 +371,7 @@ pub trait MontConfig<const N: usize>: 'static + Sync + Send + Sized {
     #[unroll_for_loops(12)]
     #[allow(clippy::modulo_one)]
     fn into_bigint(a: Fp<MontBackend<Self, N>, N>) -> BigInt<N> {
-        let mut tmp = a.0;
-        let mut r = tmp.0;
+        let mut r = a.0 .0;
```

Comment:
why the whitespace?

Reply:
the whitespace is there for the same reason we had the variable: rust thinks `a.0.0` is actually `a.(0.0)`, and parse the latter part as a float

Reply:
gosh

Reply:
I think it's rust fmt stuff. I found it strange at the beginning but I found a lots of related issues in [rustfmt](https://github.com/rust-lang/rustfmt) repo. Like this one: https://github.com/rust-lang/rustfmt/issues/5745

Reply:
this seems to work on play.rustlang tho
```

struct B (pub u8, pub u8);
struct A (pub B, pub B);

fn main() { 
    let a = A(B(0, 1), B(2, 3));
    println!("{}", a.0.0)
}
```